notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -38,12 +38,13 @@
     }
```

Comment:
<p>can we use "metrics" instead of "metric_pipeline" here? I not sure what additional info we get from "_pipeline" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>